esp/src/src/Utility.ts

```diff
@@ -209,6 +210,38 @@ export function espSkew2NumberTests() {
     }, this);
 }
 
+export function formatAsDelim(grid, rows: any, delim = ",") {
```

Comment:
Couldn't `hpccjs.common.csvFormat` be used here?

Reply:
I don't see a csvFormat in common?